app/views/widgets/show.html.erb

```diff
@@ -0,0 +1,54 @@
+<head>
+  <%= stylesheet_link_tag "widget" %>
+</head>
+<body>
+  <div class="alaveteli-widget">
+    <div class="alaveteli-widget__top">
+      <div class="alaveteli-widget__left">
+        <div class="alaveteli-widget__title">
+          <%= link_to @info_request.title, request_path(@info_request), :target => "_top" %>
+        </div>
+        <div class="alaveteli-widget__status <%= @info_request.calculate_status %>">
+          <p class="alaveteli-widget__status__status-label">Status</p>
+          <%= status_description(@info_request) %>
```

Comment:
Just noticed we actually assign the status in the controller https://github.com/mysociety/alaveteli/blob/add-widget/app/controllers/widgets_controller.rb#L17